requirements.prod.txt

```diff
@@ -41,14 +35,10 @@ presto-python-client==0.7.0
     # via opensafely-cohort-extractor (setup.py)
 py==1.10.0
     # via retry
-pyarrow==3.0.0
+pyarrow==7.0.0
```

Comment:
Did you mean to bump this?

Reply:
No, was accidental, I tried removing it but it was needed by some code
 will revert